scripts/test/Elasticsearch/TicketCreate.t

```diff
@@ -51,6 +52,27 @@ my $Helper = $Kernel::OM->Get('Kernel::System::UnitTest::Helper');
 my $UserID   = 1;
 my $RandomID = $Helper->GetRandomID();
 
+# set necessary sysconfig setting
```

Comment:
I think that the config setting changes should be temporary, using `Kernel::System::UnitTest::Helper::ConfigSettingChange()`.

Reply:
In the meantime, I removed setting 'Elasticsearch::Active', because `Kernel::System::Elasticsearch::InitialSetup()` handles this. I needed to include various other config setting adaptions, which I implemented using `ConfigSettingChange()`.